### PR TITLE
Add G Suite Training/Synergyse

### PR DESCRIPTION
**Pull request by tech234a**

### CODE REVIEW

File: graveyard.json [hunk 1]
```diff
@@ -1414,5 +1414,13 @@
     "link": "https://support.google.com/translatortoolkit/answer/9462068",
     "name": "Google Translator Toolkit",
     "type": "service"
+  },
+  {
+    "dateClose": "2019-09-30",
+    "dateOpen": "2013-04-18",
+    "description": "G Suite Training/Synergyse provided interactive and video-based training for 20 Google G Suite products in 9 languages. It was acquired by Google on May 1, 2016.",
```
Comment (codyogden) [inline graveyard.json]: Single sentence, please. Maybe change to:

> G Suite Training (aka Synergyse) was a Chrome extension that provided customizable interactive and video-based training for G Suite products in nine languages.

File: graveyard.json [hunk 2]
```diff
@@ -1414,5 +1414,13 @@
...
+    "link": "https://support.google.com/a/answer/7039472",
+    "name": "G Suite Training/Synergyse",
```
Comment (codyogden) [inline graveyard.json]: Just `G Suite Training` for the name. We should try to publish the name as the product was identified at Google.

**Comment by tech234a** [comment]
I updated my PR. I had originally had "/Synergyse" in the name because, for some reason, the address for the website remained https://portal.synergyse.com/, even though the product was re-branded.

**Comment by codyogden** [comment]
Awesome! Thank you! 🥇 💯 🙌

**Pull request merged**